### ISSUE TITLE
add frida

### ISSUE THREAD

**Issue by alvarorichard** [mentebinaria/retoolkit#97]
add frida  for instrumentation toolkit 

[https://github.com/frida/frida?tab=readme-ov-file](url)
[https://frida.re/](url)

**Comment by merces** [comment]
Good idea. Alternatively, `python -m pip install frida-tools` does the job too. ;)

**Comment by alvarorichard** [comment]
> Good idea. Alternatively, `python -m pip install frida-tools` does the job too. ;)

I agree with you, but it’d be even better if it was added to Retoolkit — it’d make the user experience a lot nicer

**Comment by merces** [comment]
Yeah, I'll work on it. 👍

**Comment by merces** [comment]
Closed with https://github.com/mentebinaria/retoolkit/commit/f01be9a35a3a376139b8a977918fb0ca1a3f4aaf
